source/index.html

```diff
@@ -1,12 +1,157 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html  class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Кэт энерджи</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+  <body class="page__body">
+    <header class="main-header">
+      <a class="main-header__logo">
+        <img src="img/logo-desktop-1.svg" width="202" height="58" alt="Кэт Энерджи">
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list site-list">
+            <li class="site-list__item site-list__item--active">
+              <a href="#">
+                Главная
+              </a>
+            </li>
+            <li class="site-list__item"><a href="#">Каталог продукции</a></li>
+            <li class="site-list__item"><a href="#">Подбор программы</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="main-container">
+      <h1 class="visually-hidden">Функциональное
+        питание для котов</h1>
+        <section class="program container">
+          <h2 class="program__title">Функциональное
+            питание для котов</h2>
```

Comment:
![повторения](https://wompampsupport.azureedge.net/fetchimage?siteId=7575&v=2&jpgQuality=100&width=700&url=https%3A%2F%2Fi.kym-cdn.com%2Fphotos%2Fimages%2Fnewsfeed%2F000%2F947%2F551%2F11c.jpg)

Ничего страшного не случится, если сделать так
```suggestion
        <section class="program container">
          <h1 class="program__title">Функциональное питание для котов</h1>
```
Наоборот пользователи СР не будут путаться об два одинаковых заголовка